src/runtime/runtime.js

```diff
@@ -817,6 +820,11 @@ export class ConfiguredRuntime {
             error = 'publisherProvidedId must be a string, value: ' + value;
           }
           break;
+        case 'useArticleEndpoint':
+          if (!isBoolean(value)) {
+            error = 'Unknown useArticleEndpoint value: ' + value;
```

Comment:
```suggestion
            error = 'useArticleEndpoint must be a boolean, type: ' + typeof value;
```

Reply:
This seems clearer, in case someone passes "true", like in your test case

Reply:
Done. I wanted to keep the message consistent with other bool params, so I'll update the others.